config.ts

```diff
@@ -29,20 +29,23 @@ const parseLogLevel = (envValue) => {
 };
 
 const parseApiSecret = () => {
-  const valid = process.env.hasOwnProperty('API_SECRET')
-    && (typeof process.env.API_SECRET === 'string')
-    && (process.env.API_SECRET.length > 0);
+  const valid = process.env.hasOwnProperty('FOVEA_BILLING_SECRET_KEY')
+    && (typeof process.env.FOVEA_BILLING_SECRET_KEY === 'string')
+    && (process.env.FOVEA_BILLING_SECRET_KEY.length > 0);
 
   if (!valid)
-    throw new Error('API_SECRET must be non-empty string');
+    throw new Error('FOVEA_BILLING_SECRET_KEY must be non-empty string');
 
-  return process.env.API_SECRET;
+  return process.env.FOVEA_BILLING_SECRET_KEY;
 };
```

Comment:
This change is dangerous!

- `API_SECRET` is used across all ganomede modules to access administrator-only endpoints. This guarantees that the caller is either another ganomede module or some that knows what he does. In particular, as you might have seen in the authentication middleware, it can be used to "spoof" authentication (put the API_SECRET followed by the username to pretend making the request as this user).
- `API_SECRET` should stay absolutely distinct from `FOVEA_BILLING_SECRET_KEY` (and both should still exist, they are different things)
- `FOVEA_BILLING_SECRET_KEY` is used to make request to Fovea.Billing, and ensure the webhook is only called by Fovea.Billing.
- In the authentication middleware: `ganomede.secretMatches` meaning shouldn't change: it's should always means `API_SECRET`, not `FOVEA_BILLING_SECRET_KEY`.

I testing the `password` field be equal to fovea.billing's secret key will only be necessary in the webhook, no need for a dedicated middleware (that won't be reused anyway).